docs/changes.rst

```diff
@@ -153,7 +153,8 @@ Released: not yet
   (See issue #494)
 
 * Added support for adjusting the width of any help output to the terminal
-  width. (See issue #518)
+  width. The width can be set using the PYWBEMCLI_WIDTH env var.
```

Comment:
Suggestion. TERMWIDITH rather than WIDITH

Reply:
Misspelled widrh

Reply:
DONE - Changed PYWBEMCLI_WIDTH to PYWBEMCLI_TERMWIDTH.